layouts/shortcodes/committers.html

```diff
@@ -0,0 +1,26 @@
+{{ $urlPre := "https://projects.apache.org" }}
+{{ $path :=  "/json/foundation" }}
+{{ $group :=  getJSON $urlPre $path "/groups.json"}}
```

Comment:
Why not even simpler?

```suggestion
{{ $groups :=  getJSON "https://projects.apache.org/json/foundation/groups.json"}}
```